diff -Nur deluge-torrent-0.5.8.orig/setup.py deluge-torrent-0.5.8.pld/setup.py
--- deluge-torrent-0.5.8.orig/setup.py	2007-12-29 10:55:55.000000000 -0700
+++ deluge-torrent-0.5.8.pld/setup.py	2007-12-30 18:17:04.000000000 -0700
@@ -150,7 +150,7 @@
     librariestype = [boost_filesystem, boost_date_time,
             boost_thread, 'z', 'pthread', 'ssl']
     
-    removals = ['-g', '-Wstrict-prototypes']
+    removals = ['-Wstrict-prototypes']
 
     if python_version == '2.5':
         cv_opt = sysconfig.get_config_vars()["CFLAGS"]
