diff -burNp deluge-1.0.5/setup.py deluge-1.0.5-new/setup.py
--- deluge-1.0.5/setup.py	2008-11-09 17:44:40.000000000 +0100
+++ deluge-1.0.5-new/setup.py	2008-11-12 16:39:58.129552094 +0100
@@ -180,18 +180,18 @@ for source in _sources:
             break
 
 _ext_modules = []
-if windows_check() or not os.path.exists(os.path.join(sysconfig.get_config_var("LIBDIR"), "libtorrent-rasterbar.so.1")):
-    # There isn't a system libtorrent library, so let's build the one included with deluge
-    libtorrent = Extension(
+# Force to build libtorrent module that comes with deluge, otherwise
+# it won't work.
+libtorrent = Extension(
         'libtorrent',
         extra_compile_args = _extra_compile_args,
         include_dirs = _include_dirs,
         libraries = _libraries,
         library_dirs = _library_dirs,
         sources = _sources
-    )
+)
 
-    _ext_modules = [libtorrent]
+_ext_modules = [libtorrent]
 
 class build_trans(cmd.Command):
     description = 'Compile .po files into .mo files'
@@ -294,7 +294,7 @@ setup(
             deluged = deluge.main:start_daemon
     """,
     ext_package = "deluge",
-    ext_modules = [libtorrent],
+    ext_modules = _ext_modules,
     fullname = "Deluge Bittorent Client",
     include_package_data = True,
     license = "GPLv3",
